packages/utils/typescript/lib/utils/get-config-path.js

```diff
@@ -15,7 +15,7 @@ const DEFAULT_TS_CONFIG_FILENAME = 'tsconfig.json';
  *
  * @return {string | undefined}
  */
-module.exports = (dir, { filename = DEFAULT_TS_CONFIG_FILENAME, ancestorsLookup = false } = {}) => {
+module.exports = (dir = "", { filename = DEFAULT_TS_CONFIG_FILENAME, ancestorsLookup = false } = {}) => {
```

Comment:
Even though path.resolve output the same for "" and ".", could you put the second option as the default value so we're explicit about the default directory for getConfigPath ?